ethereum/deploy/001_exchangeIssuanceZeroEx.ts

```diff
@@ -0,0 +1,57 @@
+import "module-alias/register";
+
+import { HardhatRuntimeEnvironment as HRE } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+import {
+  prepareDeployment,
+  findDependency,
+  getContractAddress,
+  getCurrentStage,
+  saveContractDeployment,
+  stageAlreadyFinished,
+  trackFinishedStage
+} from "@utils/index";
+
+import { DEPENDENCY } from "../deployments/utils/dependencies";
+import { CONTRACT_NAMES } from "../deployments/constants/001_exchangeIssuanceZeroEx";
+
+const {
+  WETH,
+  CONTROLLER,
+  ZERO_EX_EXCHANGE
+} = DEPENDENCY;
+
+const CURRENT_STAGE = getCurrentStage(__filename);
+
+const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (hre: HRE) {
+  const { deploy, deployer } = await prepareDeployment(hre);
+
+  const contractName = CONTRACT_NAMES.EXCHANGE_ISSUANCE_ZEROEX;
+
+  const wethAddress = await findDependency(WETH);
```

Comment:
Minor nit, I didn't note before. We should chain `const` for [brevity]( https://stackoverflow.com/questions/40319891/why-cant-i-declare-multiple-const-in-javascript):

```js
const wethAddress = await findDependency(WETH),
      controllerAddress = await findDependency(CONTROLLER);
```

Reply:
Whoah! Didn't know this was possible. Awesome.